src/nav-options.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import singleSpaReact from "single-spa-react";
+import {
+  HeaderPanel,
+  Switcher,
+  SwitcherItem,
+  SwitcherDivider
+} from "carbon-components-react/es/components/UIShell";
+
+export default singleSpaReact({
+  React,
+  ReactDOM,
+  rootComponent: AppNavButton
+});
+
+export function AppNavButton() {
+  const SourceFiles = {
```

Comment:
No

Reply:
@FlorianRappl please I'm trying to add extension slots. Please help me out. I'm really worried right now. What I doing wrong? Please can you state the changes clearly so I learn ?

I created to extensions and extensions slots as you would notice.

Reply:
I have been on this ticket for a while.

Reply:
@joachim-junior as I wrote please first have a look at other code. I have a feeling that you

- never looked at other code (neither from this nor from any other repository)
- did not run the solution on your local machine

Otherwise, you would see that it does not run and also has many problems.

Here, its really about the basics. For instance, understanding *why* we don't want absolute URLs (no, we actually *cannot* have them as they would not work).

Reply:
Sir the solution is running locally on my laptop. The problem now is when I used absolute URLs, the Solution was working well. Now I added an extension slot for available options in the nav dropdown. Now I can't see the items. That's why I wanted you to review and tell me where I went wrong.

I used your conversion [Here](https://github.com/openmrs/openmrs-esm-primary-navigation/pull/51) with @ujjwaltyagi355 as reference guide.